src/content/docs/en/guides/content-collections.mdx

```diff
@@ -79,15 +79,15 @@ export const collections = { blog, dogs };
 
 ### Defining the collection `loader`
 
-The Content Layer API allows you to fetch your content (whether stored locally in your project or remotely) and uses a `loader` property to retrieve your data. 
+The Content Layer API allows you to fetch your content (whether stored locally in your project or remotely) and uses a `loader` property to retrieve your data. A loader returns an array or object of entries each containing an `id` property that will provide a URL slug as well as an identifier to allow you to [query entries](#querying-collections). 
```

Comment:
I wouldn't include the mention of URL slug here. There's nothing specifically slug-related.

Reply:
🫡